build_config/Linux.intel.default/build_rules.mk

```diff
@@ -40,6 +40,7 @@ ESMF_F90DEFAULT         = mpif90
 ESMF_F90LINKLIBS       += -lmpi++
 ESMF_CXXDEFAULT         = mpicxx
 ESMF_CDEFAULT           = mpicc
+ESMF_MPILAUNCHOPTIONS   = -p "%g: "
```

Comment:
@jedwards4b can you please revert the changes to this file

Reply:
I would prefer not to - this is a really useful feature of mpt.

Reply:
Does it print the rank number in stdout?

Reply:
Yes

Reply:
Okay then, let's leave it in.